sdk/samples/iot/iot_samples_common.h

```diff
@@ -45,7 +45,7 @@
   { \
     (void)printf("\t\t%s ", span_description); \
     char* buffer = (char*)az_span_ptr(span); \
-    for (int32_t i = 0; i < az_span_size(span); i++) \
+    for (int32_t j = 0; j < az_span_size(span); j++) \

```

Comment:
What motivated this change?

Reply:
it expanded inside another for-loop that used an "i".  Can't define different parameters with the same name in the same scope.